node/rpc.go

```diff
@@ -0,0 +1,30 @@
+package node
+
+import (
+	"reflect"
+	"unicode"
+)
+
+// formatName converts to first character of name to lowercase.
+func formatName(name string) string {
```

Comment:
I think that this function could have a more descriptive name, though I personally appreciate the mystery of `formatName`

```suggestion
func firstCharToLower(name string) string {
```